module/User/src/User/Model/NewUser.php

```diff
@@ -72,6 +79,16 @@ public function getLidnr()
         return $this->lidnr;
     }
 
+    /**
+     * Set the user's membership number.
+     *
+     * @param int $lidnr
+     */
+    public function setLidnr($lidnr)
+    {
+        $this->lidnr = $lidnr;
+    }
+
```

Comment:
Why all this useless rearrangement of methods? It completely obscures whatever you're actually changing.